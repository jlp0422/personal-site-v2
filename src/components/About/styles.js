import styled from '@emotion/styled'

export const Copy = styled.p`
  font-size: 2rem;
  padding: 0.5rem 0;
`

export const ContactList = styled.div`
  > div {
    font-family: ${({ theme }) => theme.fonts.primary};
    font-size: 1.8rem;
  }
  span,
  a {
    font-size: 1.8rem;
  }
  a {
    cursor: pointer;
  }
`

export const ContentWrapper = styled.div`
  display: grid;
  grid-template-columns: 1fr 350px;
  grid-column-gap: 3%;
  @media only screen and (max-width: 800px) {
    grid-template-columns: 1fr 250px;
  }
  @media only screen and (max-width: 640px) {
    display: block;
  }
`

export const ImageWrapper = styled.div`
  @media only screen and (max-width: 640px) {
    width: 300px;
    margin: 2rem auto;
  }
`
