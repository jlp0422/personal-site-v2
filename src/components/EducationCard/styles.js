import styled from '@emotion/styled'
import {
  CardFlipBack,
  CardFlipContainer,
  CardFlipInner,
  CardShared,
  ExpInfo,
  ExpTitle
} from '../shared'

export const Container = styled(CardFlipContainer)`
  text-align: center;
  @media only screen and (max-width: 640px) {
    height: 250px;
  }
`

export const InnerContainer = styled(CardFlipInner)`
  box-shadow: ${({ theme }) => theme.palette.boxShadow.dark};
`

export const CardFront = styled(CardShared)`
  background-color: ${({ theme }) => theme.palette.black.darkest};
`

export const CardBack = styled(CardFlipBack)`
  background-color: ${({ theme }) => theme.palette.white.lighter};
`

export const Title = ExpTitle

export const Info = styled(ExpInfo)`
  width: 90%;
  grid-template-columns: repeat(2, 1fr);
`
