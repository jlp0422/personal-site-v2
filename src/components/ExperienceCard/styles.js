import styled from '@emotion/styled'
import {
  ExpInfo,
  ExpTitle,
  CardFlipContainer,
  CardFlipInner,
  CardFlipBack,
  CardShared
} from '../shared'

export const Container = styled(CardFlipContainer)`
  display: grid;
  text-align: center;
  margin: 2.5rem auto;
  @media only screen and (max-width: 640px) {
    height: 250px;
  }
`

export const InnerContainer = styled(CardFlipInner)`
  box-shadow: ${({ theme }) => theme.palette.boxShadow.light}
`

export const CardFront = styled(CardShared)`
  padding: 1.5rem;
  background-color: ${({ theme }) => theme.palette.black.darkest};
`

export const CardBack = styled(CardFlipBack)`
  padding: 1.5rem;
  background-color: ${({ theme }) => theme.palette.white.lighter}
`

export const Title = ExpTitle

export const Info = styled(ExpInfo)`
  width: 100%;
  align-items: center;
  @media only screen and (max-width: 640px) {
    grid-template-columns: 1fr;
  }
`

export const LogoContainer = styled.div`
  display: block;
  visibility: visible;
  width: 60px;
  margin: 1rem auto 0;
  @media only screen and (max-width: 640px) {
    display: none;
    visibility: hidden;
  }
`
