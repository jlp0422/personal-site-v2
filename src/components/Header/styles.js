import styled from '@emotion/styled'
import { Link } from 'gatsby'
import { ActiveLink } from '../../components/shared/Styled'

export const StyledHeader = styled.header`
  background: ${({ theme }) => theme.palette.link.secondary};
  margin-bottom: 1.5rem;
`

export const Div = styled.div`
  margin: 0 auto;
  max-width: 1200px;
  padding: 2rem 1.2rem;
  display: grid;
  grid-template-columns: 120px 120px 1fr 120px 120px;
  justify-items: center;
  align-items: center;
  grid-column-gap: 2rem;
  @media only screen and (max-width: 640px) {
    padding: 0.8rem;
    display: flex;
    align-items: center;
    justify-content: center;
  }
`

export const H1 = styled.h1`
  margin: 0;
  text-align: center;
  text-transform: uppercase;
`

export const HeaderLink = styled(Link)`
  letter-spacing: 0.3rem;
  font-family: ${({ theme }) => theme.fonts.header};
  color: white;
  text-decoration: none;
  font-size: 4.5rem;
  @media only screen and (max-width: 800px) {
    visibility: hidden;
    display: none;
  }
`

export const LogoLink = styled(Link)`
  margin: 0;
  visibility: hidden;
  display: none;
  @media only screen and (min-width: 640px) and (max-width: 800px) {
    visibility: visible;
    display: block;
  }
`

export const NavLink = styled(ActiveLink)`
  @media only screen and (max-width: 640px) {
    display: none;
    visibility: hidden;
  }
`

export const MobileLogo = styled.button`
  margin: 0;
  visibility: hidden;
  display: none;
  @media only screen and (max-width: 639px) {
    visibility: visible;
    display: block;
  }
`

export const List = styled.ul`
  padding: 0 0 1rem;
  list-style: none;
  margin-top: 0;
  visibility: hidden;
  display: none;
  @media only screen and (max-width: 640px) {
    visibility: visible;
    display: block;
  }
`
