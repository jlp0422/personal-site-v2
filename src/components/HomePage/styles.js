import { keyframes } from '@emotion/core'
import styled from '@emotion/styled'
import { Link } from 'gatsby'

export const spin = keyframes`
  from {
    transform: rotate(0deg)
  }

  to {
    transform: rotate(360deg)
  }
`

export const GridContainer = styled.div`
  display: grid;
  grid-template: repeat(2, 1fr) / repeat(2, 1fr);
  grid-gap: 2rem 2rem;
  justify-content: center;
  align-items: center;
  text-align: center;
  margin-top: 1rem;
  > * {
    margin: 0;
  }
  @media only screen and (max-width: 800px) {
    grid-template: repeat(4, 1fr) / 1fr;
  }
`

export const LinkBox = styled(Link)`
  background-color: ${({ color }) => color};
  padding: 8rem;
  font-family: ${({ theme }) => theme.fonts.nav};
  text-decoration: none;
  transition: all 0.2s linear;
  > h3 {
    margin: 1.5rem;
  }
  @media only screen and (min-width: 641px) {
    :hover {
      transform: scale(1.05);
      div {
        animation: ${spin} 2s linear infinite;
      }
    }
  }
  @media only screen and (max-width: 800px) {
    padding: 5rem;
  }
  @media only screen and (max-width: 640px) {
    padding: 1rem;
    > h3 {
      margin: 0.5rem;
    }
  }
`
