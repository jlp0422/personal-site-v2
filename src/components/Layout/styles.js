import styled from '@emotion/styled'

export const ContainerDiv = styled.div`
  margin: 0 auto;
  max-width: 1200px;
  padding: 0 1.5rem 1.5rem;
  min-height: 85vh;
  display: grid;
  grid-template-rows: 1fr auto;
`
