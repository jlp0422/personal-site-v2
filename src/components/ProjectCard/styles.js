import styled from '@emotion/styled'

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  background-color: white;
  margin: 1rem 0;
  padding: 1.5rem;
  border: 1px solid black;
  transition: all 0.2s linear;
  --webkit-transition: all 0.2s linear;
  :hover {
    box-shadow: ${({ theme }) => theme.palette.boxShadow.primary};
    transform: translate(-8px, -8px);
    --webkit-transform: translate(-8px, -8px);
  }
  @media only screen and (max-width: 480px) {
    box-shadow: ${({ theme }) => theme.palette.boxShadow.primary};
    :hover {
      transform: none;
    }
  }
`

export const LinkContainer = styled.div`
  display: flex;
  justify-content: space-between;
  flex-wrap: wrap;
  * + * {
    margin-top: 0;
  }
`
