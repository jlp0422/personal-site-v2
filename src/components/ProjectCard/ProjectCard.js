import React from 'react'
import { linkExists, upper, joinAndLower } from '../../helpers'
import { GithubLink, LiveLink } from '../shared'
import { Container, Description, LinkContainer, Title } from './styles'
import Image from '../Image'

const isStackjack = imageKey => imageKey === 'stackjack'

const ProjectCard = ({ project }) => {
  const imageKey = joinAndLower(project.title)
  const hasGithub = linkExists(project.github)
  const hasWebsite = linkExists(project.website)
  return (
    <Container>
      <Title>{upper(project.title)}</Title>
      <Description>{project.description}</Description>
      <div>
        {project.details.map(detail => (
          <p key={detail}>{detail}</p>
        ))}
        <Image
          queryKey={imageKey}
          style={{ maxHeight: '250px' }}
          imgStyle={{
            ...(isStackjack(imageKey)
              ? { objectPosition: '10% 10%' }
              : { objectPosition: 'top' })
          }}
        />
      </div>
      <LinkContainer>
        {hasGithub && (
          <GithubLink
            url={project.github.link}
            linkText={project.github.text}
          />
        )}
        {hasWebsite && (
          <LiveLink
            url={project.website.link}
            linkText={project.website.text}
          />
        )}
      </LinkContainer>
    </Container>
  )
}

export default ProjectCard
