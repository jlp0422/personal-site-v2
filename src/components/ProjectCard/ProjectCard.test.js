import React from 'react'
import renderer from 'react-test-renderer'
import ProjectCard from './ProjectCard'
import { ThemeWrapper, GithubLink, LiveLink } from '../shared'
import { Title } from './styles'

const DEFAULT_PROJECT = {
  title: 'Hot Tub 2019',
  github: { link: '/to/my/github', text: 'github' },
  website: { link: '/to/my/website', text: 'website' },
  details: ['i made this', 'all by myself']
}

describe('ProjectCard', () => {
  const getOutput = args => {
    const defaultProps = {
      project: DEFAULT_PROJECT
    }

    const props = { ...defaultProps, ...args }

    return renderer.create(
      <ThemeWrapper>
        <ProjectCard {...props} />
      </ThemeWrapper>
    )
  }

  it('Renders correctly', () => {
    const tree = getOutput().toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('Displays the project title in all uppercase', () => {
    const instance = getOutput().root
    expect(instance.findByType(Title).props.children).toBe('HOT TUB 2019')
  })

  it('Displays the array of details', () => {
    const instance = getOutput().root
    const details = instance.findAllByType('p')
    const getChildAtIndex = index => details[index].props.children
    expect(details).toHaveLength(2)
    expect(getChildAtIndex(0)).toBe('i made this')
    expect(getChildAtIndex(1)).toBe('all by myself')
  })

  describe('GithubLink', () => {
    it('Displays the link if it exists', () => {
      const instance = getOutput().root
      const link = instance.findByType(GithubLink)
      expect(link).toBeTruthy()
      expect(link.props).toHaveProperty('url')
      expect(link.props).toHaveProperty('linkText')
    })

    it('Does not display if no link exists', () => {
      const instance = getOutput({
        project: { ...DEFAULT_PROJECT, github: {} }
      }).root
      const link = instance.findAllByType(GithubLink)
      expect(link).toHaveLength(0)
    })
  })

  describe('LiveLink', () => {
    it('Displays the link if it exists', () => {
      const instance = getOutput().root
      const link = instance.findByType(LiveLink)
      expect(link).toBeTruthy()
      expect(link.props).toHaveProperty('url')
      expect(link.props).toHaveProperty('linkText')
    })

    it('Does not display if no link exists', () => {
      const instance = getOutput({
        project: { ...DEFAULT_PROJECT, website: {} }
      }).root
      const link = instance.findAllByType(LiveLink)
      expect(link).toHaveLength(0)
    })
  })
})
