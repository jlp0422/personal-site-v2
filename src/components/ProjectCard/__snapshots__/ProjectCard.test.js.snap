// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectCard Renders correctly 1`] = `
<div
  className="css-7qntd6"
>
  <h3
    className="css-zwll95"
  >
    HOT TUB 2019
  </h3>
  <div>
    <p>
      i made this
    </p>
    <p>
      all by myself
    </p>
    <div
      className=" gatsby-image-wrapper"
      style={
        Object {
          "maxHeight": "250px",
          "overflow": "hidden",
          "position": "relative",
        }
      }
    >
      <div
        aria-hidden={true}
        style={
          Object {
            "paddingBottom": "75.5%",
            "width": "100%",
          }
        }
      />
      <img
        alt=""
        aria-hidden={true}
        src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAPCAYAAADkmO9VAAAACXBIWXMAAAsSAAALEgHS3X78AAACQElEQVQ4y42SSXPTQBCF9Xs5cuEXcOdKLlzgkDM3lhTEKQoqUCSx9s3aR7ISh8S2ZHnRo3tsGS9VgKq65lnu/vr1tBTHG+BGNaBqBlx/gMVyieWqxYKCz7ZtwU93/utRTl6/xbPnJ3j55gPKCTBerGNCUS8hoURD+59QpSiGECJDEVwgMd9j+Usjyj3QFEQYbRN3nf5NK3eje4wfR2jDFxA/nyBVn0LEp7i7eYV5/xTzOEQ7a9ZFq/aowaFWqqpCXuS4va1gPwj08mv0yivkeh+L3g9UZ+eYiRzz1UoW1HWN2WwmNdc287nUTdNgRTkKJ/R9ge+6j4fRHWbTCvV4igklP9YVpos5kkzA932CUVPHkZphlmUjjmOpDdNEWZZQckGutBQftQKOHyIIfPiDAQYUAW09DAI4BLEsC1EUSe15HmLS/I41v9d1nWoDKGU5RE8XOKfwghhpEsuuXaRpKovYjciyLTAjbdu2bMzaJIcMVsrhEJ+0TEIZmBAkitYwTkiSRAK4uIPzyKzZIWsG6oaBMAzpsyHgNzPFpZUiCDeg+BjIzhiy61COvIFvHTKwp2d418/hDBgUyT+6YKDrutJhJl3ZBHUlkBfRwTVNWzscEvArOTxTBVSXRz4ExnB3RuaTG+yOzJqXsgEW+GKktJQMfngM5NH/3GGyd4dm55A0OwwYyEu5INhnTeDaSfY23I3M92aa1naz7PBw5L6qrj+boihwaSa4Ipjh7bvjEbYOCdrB+Tdri+Ce50ttbLb8G7OYUmWas1qgAAAAAElFTkSuQmCC"
        style={
          Object {
            "height": "100%",
            "left": 0,
            "margin": 0,
            "objectFit": "cover",
            "objectPosition": "top",
            "opacity": 1,
            "position": "absolute",
            "top": 0,
            "transitionDelay": "500ms",
            "width": "100%",
          }
        }
        title="hot-tub-2019"
      />
      <picture>
        <source
          sizes="(max-width: 800px) 100vw, 800px"
          srcSet="/static/8f3a0f809cfd0972b2855092b9151ef2/61e93/hot-tub-2019.webp 200w,
/static/8f3a0f809cfd0972b2855092b9151ef2/1f5c5/hot-tub-2019.webp 400w,
/static/8f3a0f809cfd0972b2855092b9151ef2/58556/hot-tub-2019.webp 800w,
/static/8f3a0f809cfd0972b2855092b9151ef2/c515d/hot-tub-2019.webp 1191w"
          type="image/webp"
        />
        <source
          sizes="(max-width: 800px) 100vw, 800px"
          srcSet="/static/8f3a0f809cfd0972b2855092b9151ef2/69585/hot-tub-2019.png 200w,
/static/8f3a0f809cfd0972b2855092b9151ef2/497c6/hot-tub-2019.png 400w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ee604/hot-tub-2019.png 800w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ef8da/hot-tub-2019.png 1191w"
        />
        <img
          alt="hot-tub-2019"
          loading="lazy"
          onLoad={[Function]}
          sizes="(max-width: 800px) 100vw, 800px"
          src="/static/8f3a0f809cfd0972b2855092b9151ef2/ee604/hot-tub-2019.png"
          srcSet="/static/8f3a0f809cfd0972b2855092b9151ef2/69585/hot-tub-2019.png 200w,
/static/8f3a0f809cfd0972b2855092b9151ef2/497c6/hot-tub-2019.png 400w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ee604/hot-tub-2019.png 800w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ef8da/hot-tub-2019.png 1191w"
          style={
            Object {
              "height": "100%",
              "left": 0,
              "margin": 0,
              "objectFit": "cover",
              "objectPosition": "top",
              "opacity": 0,
              "position": "absolute",
              "top": 0,
              "transition": "opacity 500ms",
              "width": "100%",
            }
          }
          title="hot-tub-2019"
        />
      </picture>
      <noscript
        dangerouslySetInnerHTML={
          Object {
            "__html": "<picture><source type='image/webp' srcset=\\"/static/8f3a0f809cfd0972b2855092b9151ef2/61e93/hot-tub-2019.webp 200w,
/static/8f3a0f809cfd0972b2855092b9151ef2/1f5c5/hot-tub-2019.webp 400w,
/static/8f3a0f809cfd0972b2855092b9151ef2/58556/hot-tub-2019.webp 800w,
/static/8f3a0f809cfd0972b2855092b9151ef2/c515d/hot-tub-2019.webp 1191w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><source srcset=\\"/static/8f3a0f809cfd0972b2855092b9151ef2/69585/hot-tub-2019.png 200w,
/static/8f3a0f809cfd0972b2855092b9151ef2/497c6/hot-tub-2019.png 400w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ee604/hot-tub-2019.png 800w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ef8da/hot-tub-2019.png 1191w\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 800px) 100vw, 800px\\" srcset=\\"/static/8f3a0f809cfd0972b2855092b9151ef2/69585/hot-tub-2019.png 200w,
/static/8f3a0f809cfd0972b2855092b9151ef2/497c6/hot-tub-2019.png 400w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ee604/hot-tub-2019.png 800w,
/static/8f3a0f809cfd0972b2855092b9151ef2/ef8da/hot-tub-2019.png 1191w\\" src=\\"/static/8f3a0f809cfd0972b2855092b9151ef2/ee604/hot-tub-2019.png\\" alt=\\"hot-tub-2019\\" title=\\"hot-tub-2019\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
          }
        }
      />
    </div>
  </div>
  <div
    className="css-1h5sbg5"
  >
    <div
      className="css-88m2o9"
    >
      <img
        alt="git-logo"
        height={20}
        src="/static/76d344f08412543c74ccd1fb2493d607/git-logo.svg"
        title="git-logo"
        width={20}
      />
      <div>
        <a
          className="css-e0yupj"
          href="/to/my/github"
          target="_blank"
        >
          github
        </a>
      </div>
    </div>
    <div
      className="css-88m2o9"
    >
      <div
        className=" gatsby-image-wrapper"
        style={
          Object {
            "overflow": "hidden",
            "position": "relative",
            "width": "2.2rem",
          }
        }
      >
        <div
          aria-hidden={true}
          style={
            Object {
              "paddingBottom": "100%",
              "width": "100%",
            }
          }
        />
        <img
          alt=""
          aria-hidden={true}
          src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0NAAAACXBIWXMAAAsTAAALEwEAmpwYAAABVElEQVQ4y+XUzStEURjH8Wvu3IWdnbKSEpkNpdhYKIasLGyUt2tBUshGibLxkmSBiSLMwmuxMylLK9lYWClLdv4H36d+o9PNYKa7c+pTZ86555lzzn3u43meV+bF175itWIMIUaLFGptWz5YLx6xhs0SreIBgxZwH0MxHLcLV9bZdgIGSBQp0NpunFlnxwnol7CzhBPw/B8HHInhpfTkA2bQF0PaWGJfeErMV+VQDjcROc39NH+NZxxZwD3MoxIp1Dvsdw36MaB+9JkGVGEclxZwTlttQRod0ol2NGEKs2jUmPtcWrXgEMsWMIklnGrLx8jiALd4wy628I47zWX1rK05wQrKf7voad1xvtkfL/zlDflKgaQzZvdyjwpnLNBYrTOW1Fq/UHFMqZTZUV+0ow1ZxBM+sI7mSHJ/G7BaRw31BU1iQqw/LDOoi1bqgiW8lLJv7RMGcFWdkI4KZgAAAABJRU5ErkJggg=="
          style={
            Object {
              "height": "100%",
              "left": 0,
              "margin": 0,
              "objectFit": "cover",
              "objectPosition": "center",
              "opacity": 1,
              "position": "absolute",
              "top": 0,
              "transitionDelay": "500ms",
              "width": "100%",
            }
          }
          title="computer-icon-2"
        />
        <picture>
          <source
            sizes="(max-width: 512px) 100vw, 512px"
            srcSet="/static/db0c5da11b69426f19ad7d5fb9021289/61e93/computer-icon-2.webp 200w,
/static/db0c5da11b69426f19ad7d5fb9021289/1f5c5/computer-icon-2.webp 400w,
/static/db0c5da11b69426f19ad7d5fb9021289/a3e81/computer-icon-2.webp 512w"
            type="image/webp"
          />
          <source
            sizes="(max-width: 512px) 100vw, 512px"
            srcSet="/static/db0c5da11b69426f19ad7d5fb9021289/69585/computer-icon-2.png 200w,
/static/db0c5da11b69426f19ad7d5fb9021289/497c6/computer-icon-2.png 400w,
/static/db0c5da11b69426f19ad7d5fb9021289/bc59e/computer-icon-2.png 512w"
          />
          <img
            alt="computer-icon-2"
            loading="lazy"
            onLoad={[Function]}
            sizes="(max-width: 512px) 100vw, 512px"
            src="/static/db0c5da11b69426f19ad7d5fb9021289/bc59e/computer-icon-2.png"
            srcSet="/static/db0c5da11b69426f19ad7d5fb9021289/69585/computer-icon-2.png 200w,
/static/db0c5da11b69426f19ad7d5fb9021289/497c6/computer-icon-2.png 400w,
/static/db0c5da11b69426f19ad7d5fb9021289/bc59e/computer-icon-2.png 512w"
            style={
              Object {
                "height": "100%",
                "left": 0,
                "margin": 0,
                "objectFit": "cover",
                "objectPosition": "center",
                "opacity": 0,
                "position": "absolute",
                "top": 0,
                "transition": "opacity 500ms",
                "width": "100%",
              }
            }
            title="computer-icon-2"
          />
        </picture>
        <noscript
          dangerouslySetInnerHTML={
            Object {
              "__html": "<picture><source type='image/webp' srcset=\\"/static/db0c5da11b69426f19ad7d5fb9021289/61e93/computer-icon-2.webp 200w,
/static/db0c5da11b69426f19ad7d5fb9021289/1f5c5/computer-icon-2.webp 400w,
/static/db0c5da11b69426f19ad7d5fb9021289/a3e81/computer-icon-2.webp 512w\\" sizes=\\"(max-width: 512px) 100vw, 512px\\" /><source srcset=\\"/static/db0c5da11b69426f19ad7d5fb9021289/69585/computer-icon-2.png 200w,
/static/db0c5da11b69426f19ad7d5fb9021289/497c6/computer-icon-2.png 400w,
/static/db0c5da11b69426f19ad7d5fb9021289/bc59e/computer-icon-2.png 512w\\" sizes=\\"(max-width: 512px) 100vw, 512px\\" /><img loading=\\"lazy\\" sizes=\\"(max-width: 512px) 100vw, 512px\\" srcset=\\"/static/db0c5da11b69426f19ad7d5fb9021289/69585/computer-icon-2.png 200w,
/static/db0c5da11b69426f19ad7d5fb9021289/497c6/computer-icon-2.png 400w,
/static/db0c5da11b69426f19ad7d5fb9021289/bc59e/computer-icon-2.png 512w\\" src=\\"/static/db0c5da11b69426f19ad7d5fb9021289/bc59e/computer-icon-2.png\\" alt=\\"computer-icon-2\\" title=\\"computer-icon-2\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
            }
          }
        />
      </div>
      <div>
        <a
          className="css-e0yupj"
          href="/to/my/website"
          target="_blank"
        >
          website
        </a>
      </div>
    </div>
  </div>
</div>
`;
